tools/launchdev.sh

```diff
@@ -413,6 +424,13 @@ function st2start(){
             --config-file $ST2_CONF --register-all
     fi
 
+    if [ "$copy_test_packs" = true ]; then
+        st2 run packs.setup_virtualenv packs=tests
+        if [ $? != 0 ]; then
+            echo "Warning: Please setup virtualenv for pack \"tests\" before run integration test"
```

Comment:
Please change the message to `Unable to setup virtualenv for the \"tests\" pack. Please setup virtualenv for the \"tests\" pack before running integration tests.`